### ISSUE TITLE
Frequent warnings during build on Jenkins

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#908]
on master branch we can see this often:
```
In file included from /home/jenkins/workspace/openscap-master/src/OVAL/oval_syschar.c:32:0:
/home/jenkins/workspace/openscap-master/build/config.h:11:0: warning: "NDEBUG" redefined
 #define NDEBUG
 
<command-line>:0:0: note: this is the location of the previous definition
````

**Comment by mpreisler** [comment]
Doesn't cmake set it automatically via CLI?

**Comment by jan-cerny** [comment]
Fixed in #1079
